src/ShopBundle/objects/ArticleList/DatabaseAccessLayer/SortType.php

```diff
@@ -12,16 +12,26 @@
 namespace ChameleonSystem\ShopBundle\objects\ArticleList\DatabaseAccessLayer;
 
 use ChameleonSystem\ShopBundle\objects\ArticleList\DatabaseAccessLayer\Interfaces\SortTypeInterface;
+use ChameleonSystem\CoreBundle\ServiceLocator;
+use ChameleonSystem\CoreBundle\Service\PortalDomainService;
+use ChameleonSystem\CoreBundle\Service\LanguageServiceInterface;
+use Symfony\Component\Config\Resource\SelfCheckingResourceChecker;
```

Comment:
Looks like this service slipped in? It is not used, right?

Reply:
Correct, I removed it